zproject/prod_settings_template.py

```diff
@@ -120,12 +120,43 @@
     # 'zproject.backends.GoogleAuthBackend',  # Google auth, setup below
     # 'zproject.backends.GitHubAuthBackend',  # GitHub auth, setup below
     # 'zproject.backends.GitLabAuthBackend',  # GitLab auth, setup below
+    # 'zproject.backends.AppleAuthBackend',  # Apple auth, setup below
     # 'zproject.backends.AzureADAuthBackend',  # Microsoft Azure Active Directory auth, setup below
     # 'zproject.backends.SAMLAuthBackend', # SAML, setup below
     # 'zproject.backends.ZulipLDAPAuthBackend',  # LDAP, setup below
     # 'zproject.backends.ZulipRemoteUserBackend',  # Local SSO, setup docs on readthedocs
 )  # type: Tuple[str, ...]
 
+########
+# Apple OAuth.
+#
+# To set up Sign In with Apple, you'll need to do the following:
+#
+# (1) Visit https://developer.apple.com/account/resources/,
+# Follow the instructions in
+# https://help.apple.com/developer-account/?lang=en#/dev77c875b7e
+# to create a Sign In Certificate.
```

Comment:
Let's be sure to consistently use the same terminology as Apple's docs do. The thing created through these instructions is a "private key", or a "Sign in with Apple private key", not a "certificate".

(This particular distinction is actually a very important one, beyond matching Apple's terminology. Certificates generally are not secret -- they are public things that you might share widely, showing them to anyone you want to talk to. Private keys are, well, private -- it's essential not to let anyone else see a copy of one.)